pkg/controller/migmigration/registry.go

```diff
@@ -62,7 +62,11 @@ func (t *Task) getAnnotations(client k8sclient.Client) (map[string]string, error
 			}
 		}
 	}
-	if t.quiesce() {
+	shouldQuiesce, err := t.isStorageConversionMigration()
```

Comment:
Is the intent that storage conversions *always* quiesce regardless of whether the user specified to quiesce? That seems to be what's happening here, so I just wanted to confirm. Also, assuming that's true, I'd rename the local var `shouldQuiesce` to `shouldQuiesceForStorageConversion` just to avoid confusion, since in regular migerations where the user explicitly specified quiesce, the current `shouldQuiesce` var gets a false value, which is confusing.